src/Subscription.php

```diff
@@ -285,6 +291,7 @@ public function cancelAt(Carbon $endsAt, $reason = SubscriptionCancellationReaso
             $this->fill([
                 'ends_at' => $endsAt,
                 'cycle_ends_at' => null,
+//                'next_plan' => null,
```

Comment:
Keep it (see other PR, #114 )

Reply:
Yeah, looks like an artifact from the previous PR. Removed.